src/olympia/accounts/views.py

```diff
@@ -271,3 +273,50 @@ def retrieve(self, request, *args, **kwargs):
             # address has an account associated with it.
             user = STUB_FXA_USER
         return Response(self.get_serializer(user).data)
+
+
+class AccountSuperCreate(JWTProtectedView):
+
+    @waffle_switch('super-create-accounts')
+    def post(self, request):
+        if not acl.action_allowed_user(request.user,
+                                       'Accounts', 'SuperCreate'):
+            log.info(
+                u'Access denied for super-create; user {} is not in the '
+                u'right group'.format(request.user))
+            raise exceptions.AuthenticationFailed('insufficient access group')
+
+        user_token = os.urandom(4).encode('hex')
+        username = 'super-created-{}'.format(user_token)
+        email = '{}@addons.mozilla.org'.format(username)
```

Comment:
I think @davehunt is using http://restmail.net/ to check email messages so he might need to be able to set the email address.